lib/release_notes_bot/release_tags.ex

```diff
@@ -0,0 +1,47 @@
+defmodule ReleaseNotesBot.ReleaseTags do
+  @moduledoc """
+  Embedded schema for the release tags view which is a query onto of the existing webhook events.
+  """
+
+  alias ReleaseNotesBot.Schema.ReleaseTag
+  alias ReleaseNotesBot.Repo
+
+  def sanitize(params) do
+    ReleaseTag.changeset(%ReleaseTag{}, params)
+  end
+
+  def generate(release_tag_id) do
+    Repo.query("
+      WITH we AS (
+        SELECT
+          raw_payload -> 'release' ->> 'id' as release_id,
+          raw_payload -> 'release' -> 'tag_name' as tag_name,
+          raw_payload ->> 'action' as event_type
+        FROM
+          webhook_events
+      )
+      SELECT
+        tag_name
+      FROM we
+      WHERE
+        event_type = 'released' and release_id = '#{release_tag_id}';
```

Comment:
is this doing string interpolation for this param? Is there a way to add this as a SQL query param?

Avoid SQL injection attacks

Reply:
Right now the path towards this is unclear to me. I can look into this more when I return. For now, this does work. If we need this merged, we can merge it in and make a ticket to handle this sql injection problem.